### PR TITLE
example index

### PR DESCRIPTION
**Pull request by ethanlcz**

### CODE REVIEW

**Comment by skylarCai** [comment]
Security Check : e70c32bdbd90e3ac97ebaf0b5eafc09e05204500

**Comment by skylarCai** [comment]
auto build success,number:205

**Review by wushengtws — APPROVED** [review]

**Comment by skylarCai** [comment]
Security Check : e70c32bdbd90e3ac97ebaf0b5eafc09e05204500

File: haas_lib_bundles/python/docs/examples/alcohol_detector/haaseduk1/README.md [hunk 1]
```diff
@@ -149,48 +146,34 @@
...
-> Python脚本的详细说明请参考脚本内嵌的文字版注释
+<br>
 
 1. **修改路由器名称及密码**
 
-
-修改hcho_detector工程里main.py中wifiSsid和wifiPassword的值为读者实际要连接的路由器的名称及密码（请注意名称和密码都需要放在""符号中间）。
+修改工程里main.py中wifiSsid和wifiPassword的值为读者实际要连接的路由器的名称及密码（请注意名称和密码都需要放在""符号中间）。
 
 ```python
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/alcohol_detector/haaseduk1/README.md:167]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**